optlang/gurobi_interface.py

```diff
@@ -525,13 +551,24 @@ def __init__(self, problem=None, *args, **kwargs):
             super(Model, self)._add_constraints(constraints, sloppy=True)
 
             gurobi_objective = self.problem.getObjective()
+            if self.problem.IsQP:
```

Comment:
I don't think this is actually necessary here since the objective expression is lazily retrieved from the solver when upon `objective.expression`

Reply:
So just getting rid of that does not work for some tests. Especially the ones where you pass a gurobipy.Model to the constructor...